db/revtree.go

```diff
@@ -22,13 +22,14 @@ type RevKey string
 
 // Information about a single revision.
 type RevInfo struct {
-	ID       string
-	Parent   string
-	BodyKey  string // Used when revision body stored externally (doc key used for external storage)
-	Deleted  bool
-	depth    uint32
-	Body     []byte // Used when revision body stored inline (stores bodies)
-	Channels base.Set
+	ID           string
+	Parent       string
+	BodyKey      string // Used when revision body stored externally (doc key used for external storage)
+	Deleted      bool
+	depth        uint32
+	Body         []byte // Used when revision body stored inline (stores bodies)
+	Channels     base.Set
+	IncludeV2Att bool
```

Comment:
Change to HasAttachments throughout, to indicate that this is a status, not an action.